publishable/config/voyager.php

```diff
@@ -94,21 +95,19 @@
     */
 
     'widgets' => [
-        [
-            'name'  => 'User',
+        'user' => [
             'icon'  => 'voyager-group',
             'model' => TCG\Voyager\Models\User::class,
             'url'   => 'admin/users',
             'image' => '/images/widget-backgrounds/02.png',
         ],
```

Comment:
`v0.11` no longer uses this, but uses classes like [this](https://github.com/the-control-group/voyager/blob/master/publishable/config/voyager.php#L117).

Maybe instead for specific configuration for the user, post and page dimmers, you could do something like this:
```
'dimmers' => [
    'user' => [/* ... */],
    'post' => [/* ... */],
    'page' => [/* ... */],
],
```